query/src/main/java/org/infinispan/query/impl/IndexInspector.java

```diff
@@ -61,6 +61,10 @@ public IndexInspector(Configuration cfg, SearchIntegrator searchFactory) {
       return indexManagerPerClass;
    }
 
+   public IndexManager get(Class<?> c) {
```

Comment:
This one does not seem the be used anywhere. The name is also a bit non-descriptive. Maybe we could call it getIndexManager, or getIndexManagerForClass, or remove it if really not used.

Reply:
I will double check

Reply:
This was not used. It's probably a leftover